aries_cloudagent/protocols/endorse_transaction/v1_0/manager.py

```diff
@@ -772,20 +767,17 @@ async def set_transaction_my_job(self, record: ConnRecord, transaction_my_job: s
         return tx_job_to_send
 
     async def set_transaction_their_job(
-        self, tx_job_received: TransactionJobToSend, receipt: MessageReceipt
+        self, tx_job_received: TransactionJobToSend, connection: ConnRecord
```

Comment:
I was looking at this for a bit and didn't understand why it used MessageReceipt here?

Reply:
Indeed; my impression is that it was just a question of the implementer being unfamiliar with the request context or something along those lines.